packages/aws-fargate/test/Control.js

```diff
@@ -19,10 +19,21 @@ let execArg;
 function Control(opts) {
   AbstractServerlessControl.call(this, opts);
   this.port = opts.port || portfinder();
+
+  this.backendUsesHttps = 'backendUsesHttps' in opts ? opts.backendUsesHttps : false;
+  const backendProtocol = this.backendUsesHttps ? 'https' : 'http';
+
+  // We execute requests from THIS process. If any url runs on https,
+  // we need to disable the TLS certificate check, otherwise the request will fail.
+  // Refer to the problem discussed in https://github.com/node-fetch/node-fetch/issues/1
+  if (this.backendUsesHttps) {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
We do not set NODE_TLS_REJECT_UNAUTHORIZED globally for a whole test env.
Only if the backend / app runs in https and we use self signed certs.